sns_ik_lib/matlab/snsIk_acc_rr.m

```diff
@@ -0,0 +1,122 @@
+function [ddq, s, exitCode] = snsIk_acc_rr(ddqLow, ddqUpp, ddxGoal, J, dJdq)
+% [ddq, s, exitCode] = snsIk_acc_rr(ddqLow, ddqUpp, ddxGoal, J, dJdq)
+%
+% This function implements the basic QP version of the SNS-IK velocity
```

Comment:
This function description needs to be updated.

Reply:
velocity -> acceleration

Reply:
QP needs to be removed

Reply:
The hazards of cut and paste!  I fixed it. I think.